hack/prepare-operatorhub-pr.sh

```diff
@@ -9,7 +9,7 @@
 
 TMP_OCI_PATH=$(mktemp -d out/sbo-bundle-oci.XXX)
 
-skopeo copy --src-tls-verify=false docker://$2 oci:$TMP_OCI_PATH:bundle
+skopeo copy --src-tls-verify=false --src-no-creds docker://$2 oci:$TMP_OCI_PATH:bundle
```

Comment:
how is this change related to PR?

Reply:
Without it, the hack script fails with the following:
```
./hack/prepare-operatorhub-pr.sh 0.7.1 quay.io/pmacik/sbo-pr-checks:pr-115-d724d340-bundle
time="2021-06-07T23:26:11Z" level=fatal msg="Error initializing source docker://quay.io/pmacik/sbo-pr-checks:pr-115-d724d340-bundle: error getting username and password: 1 error occurred:\n\t* error reading JSON file \"/run/containers/1001/auth.json\": open /run/containers/1001/auth.json: permission denied\n\n"
make: *** [Makefile:272: prepare-operatorhub-pr] Error 1
```